modules/cluster/addons.tf

```
@@ -59,7 +59,11 @@ module "kube_proxy" {
   config = local.config
   manifest = templatefile(
     "${path.module}/addons/kube-proxy.yaml",
-    { aws_region = data.aws_region.current.name },
+    {
+      aws_region   = data.aws_region.current.name
+      bind_address = var.support_ipv6 ? "0.0.0.0" : "127.0.0.1"
```

:exclamation: I don't think this is a great fix since we are allowing any host to connect under IPv4 as well. Seems like kube-proxy doesn't handle this configuration very gracefully!

Regardless, since we do not use IPv6 ATM and we are not supporting it (i.e., we haven't tested it at all), can we simply drop IPv6 for now? 

```suggestion
      bind_address = "127.0.0.1"
```